LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 Bonsai Foundation CIC
+Copyright (c) 2022 Bonsai Foundation CIC and Contributors
```

Comment:
Copyright year is stale. My understanding is including the year is mostly an anachronism, consider removing unless our legal counsel has recommended otherwise.

Reply:
Yeah, this is a good point, I've seen tons of people drop the year. Alternatively we can revert to just leave the date of first publication to establish a clear precedent, but I might just remove them altogether as it is getting quite tedious to update every year.